### ISSUE TITLE
Spread the word

### ISSUE THREAD

**Issue by tomek-he-him** [studio-b12/polydox#3]
@ChiperSoft @trusktr what do you guys think? Do we need this thing?

**Comment by trusktr** [comment]
@tomekwi Cool! This definitely makes the CLI easier to use. I was using the dox programmatically, so I didn't use the CLI. I skipped the CLI for that reason. hehe.

**Comment by tomek-he-him** [comment]
>  I was using the dox programmatically, so I didn't use the CLI. I skipped the CLI for that reason. hehe.

Yeah, same with me :) But that usually meant writing a doc generation script for every project.

So I’m now working on a generic, flexible tool named [_doxie_](https://github.com/studio-b12/doxie) – targeting CLI usage. Drop a short npm script into your package.json and you’re off. Tweak and polish your docs to the last detail – or take a ready-made preset.

**Comment by trusktr** [comment]
Aah, cool! Got a sample repo that uses doxie to make docs?

**Comment by tomek-he-him** [comment]
Not yet :) It’s everything a work in progress :) But I’ll keep you updated.

**Comment by tomek-he-him** [comment]
Right @trusktr now there are a couple of examples in the wild: https://github.com/studio-b12/doxie/tree/8c9150d#made-with-doxie.

One of them is this repo itself! :) Check it out: https://github.com/studio-b12/polydox/blob/c7d9c09/package.json#L17.

**Comment by trusktr** [comment]
The output is ****ing amazing.
